etc/scripts/mvn.sh

```diff
@@ -1,3 +1,11 @@
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
rename this file to something else, easy to interpret as `mvn` used to call maven

Reply:
Done